automatic/opera/tools/chocolateyUninstall.ps1

```diff
@@ -2,17 +2,32 @@
 
 $packageName         = $env:chocolateyPackageName
 $softwareNamePattern = 'Opera*'
+$toolsDir            = "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)"
 
 [array] $key = Get-UninstallRegistryKey $softwareNamePattern
 if ($key.Count -eq 1) {
   $key | ForEach-Object {
     $packageArgs = @{
       packageName    = $packageName
-      silentArgs     = "/uninstall /silent"
+      silentArgs     = "/uninstall"
       fileType       = 'EXE'
       validExitCodes = @(0)
       file           = $_.UninstallString.replace(' /uninstall', '').trim('"')
     }
+
+    $installLocation = Get-AppInstallLocation -AppNamePattern $softwareNamePattern
+    if ($null -ne $installLocation) {
+      Remove-Process -PathFilter "$([regex]::Escape($installLocation)).*" | Out-Null
+    }
```

Comment:
Stopping the process should generally happen in a `chocolateyBeforeModify.ps1` script.

Can this be moved to that file?

Reply:
I actually was considering putting it there, but opted against it when I tested the upgrade case. Turns out an upgrade can be done successfully without closing the current browser instance(s), and will take effect on the next boot. I was a little hesitant to make the package close it anyway when there's no apparent need for it to be done.

Did you still want me to move this over?

Reply:
Hmm, considering it runs successfully but just isn't fully upgraded, I am honestly not sure about what would be the appropriate way here.

@pauby and @JPRuskin, what are your thoughts on this?

Reply:
Ah, I like that behaviour - I think that upgrading in the background is ideal, though perhaps with a prompt to the user that they'll need to restart to benefit? Would vote for this being fine.